integration/spark/src/main/scala/org/apache/carbondata/indexserver/DistributedPruneRDD.scala

```diff
@@ -143,6 +143,7 @@ private[indexserver] class DistributedPruneRDD(@transient private val ss: SparkS
         val reader = dataMapFormat.createRecordReader(inputSplit, attemptContextImpl)
         reader.initialize(inputSplit, attemptContextImpl)
         while (reader.nextKeyValue()) {
+          reader.getCurrentValue.setRowCount(reader.getCurrentValue.getDetailInfo.getRowCount)
```

Comment:
I dont think this is the root cause of this issue. We need to check why CarbonInputSplit in ExtendedBlocklet has the wrong count